### ISSUE TITLE
New: Add More Time to running session

### ISSUE THREAD

**Issue by micahp2** [totem-technologies/totem#400]

**Comment by ansells** [comment]
@micahp2 If you are already in "overtime" should you be able to add time to the session? I would think not, but if you did then it would need to be added from "now" and not the session end time, right?

Also, since you say you can "remove minutes" then I assume this value can go negative, correct?

**Comment by micahp2** [comment]
@ansells here's what I would say to your questions:

- Once in overtime you can't add more time; you can only end the session
- You can add more time while still in the normal, scheduled session time
- Removing minutes from the normal scheduled time shouldn't make the session time go negative; yes the input for this option would have to show negative minutes being applied to the total time

Does that help?

**Comment by ansells** [comment]
Yep, very helpful. One last question. Once the session is in overtime should the menu button just go away? Or, perhaps we should put the "end session" as an option under that menu rather than in the dialog itself?

**Comment by micahp2** [comment]
I would switch the button to the "end session" action and show that to the Keeper. That's a better use of that space than hiding or disabling the button.
